.eslintrc.js

```diff
@@ -70,7 +70,11 @@ module.exports = {
     'no-nested-ternary': 'warn',
     'prefer-destructuring': 'warn',
     'no-unreachable': 'warn',
-    'cypress/unsafe-to-chain-command': 'warn',
+    'cypress/unsafe-to-chain-command': 'off',
```

Comment:
@dadiorchen I changed this from `"warn"` to `"off"` because I was getting **234** errors when trying to commit, let me know what your opinion is here.

Reply:
I am fine with this